yarn.lock

```diff
@@ -0,0 +1,923 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.

```

Comment:
We don't want this file. Can revert this, and keep only the changes related to README.md